client/.eslintrc.json

```diff
@@ -17,7 +17,12 @@
   },
   "plugins": ["react", "react-hooks", "prettier"],
   "rules": {
-    "prettier/prettier": "error",
+    "prettier/prettier": [
+      "error",
```

Comment:
Can you explain why we need this change?

Reply:
Can we drop these changes if they were not intentional?